### ISSUE TITLE
Sometimes delimiter change when uploading a new file during task creation is ignored

### ISSUE THREAD

**Issue by brodecva** [odalic/odalic-ui#415]
It happens only sometimes, when I change the comma to colon.

**Comment by istvan-satmari** [comment]
I am sorry, but I am unable to reproduce the bug.
Also, it seems weird that it even happens, since I checked the code - if you click "Configure" right of the selected file, select a proper delimiter and then click "Save and close", it should not be possible for the modal window to close itself, until server acknowledges the changes. 
Looking at these lines of code:
scope.save = function (future) {
              onSave(
                       // Success => close the modal
                        function (response) {
                            scope.dismiss(); // closes the modal
                            future(); // will change the clicked button from "spinner"  back to "classic button" (here it serves for further openings of the same modal, so yes, it is needed)
                        },
                        // Failure while sending data => display an error
                        function (response) {
                            scope.messages.push('error', reporth.constrErrorMsg(scope['msgtxt.saveFailure'], response.data)); // display an error to the user
                            future(); // will change...
                        }
                    );
                };

I really see no way of the modal closing before the file configuration is actually properly sent and saved on the server. The only option is the data sent do not correspond to the changes you made, but this does not seem to be the cause either.

=> Did you click on the "Save and close" button to close the modal? (no other way of closing the modal saves the changes). It is possible I am missing something here. Do let me know, thanks!

**Comment by brodecva** [comment]
Actually how I came to notices this bug is that I successfuly created the task and when I let it run, it ended almost immediately because a wrong delimiter was used. First I blamed myself, that I forgot to change it from coma to colon, but it is not the case, because even when I do mind to change it, the bug sometimes appears too. I almost suspect that it might happen in cases when I rush to create the task, not waiting for all the background AJAX calls to finish... Will look into it, but I cannot close this one, because I do not imagine it. Also maybe it happens only in Firefox...
